docs/docs/workflows/config.md

```diff
@@ -0,0 +1,228 @@
+# Configure your cluster
+
+Before your cluster can be created, you need to configure the identity and access management (IAM) for your CSP and choose machine types for your cluster.
+
+## Choosing VM types
+
+Constellation supports the following VM types:
+<tabs groupID="csp">
```

Comment:
```suggestion
<tabs groupId="csp">
```
Docusaurus seems to be case sensitive with this. At least this change fixes the tabs not being in sync.